docs/posts/blog_AnaRuvalcaba/index.qmd

```diff
@@ -19,7 +19,7 @@ program. The internship program provided a unique learning environment for nearl
 worked in interdisciplinary teams to build Jupyter extensions. 
 
 Starting in 2019, I participated in a multi-year effort to create a new governance model for Project Jupyter and am 
```

Comment:
Should "I" be added? (From "for Project Jupyter and am" to "for Project Jupyter and **I** am") ..?

Reply:
@DeaMariaLeon The current wording is within the norms of English grammar.